docs/7.x/testplan.md

```diff
@@ -7,7 +7,7 @@
 ### Hub
 
 - [ ] Install Hub in CLI mode.
-  - [ ] Verify can configure [OIDC connector](https://gravitational.com/gravity/docs/ver/6.x/config/#example-google-oidc-connector), for example:
+  - [ ] Verify can configure [OIDC connector](https://gravitational.com/gravity/docs/ver/6.x/config.md#example-google-oidc-connector), for example:
```

Comment:
Should this be a relative link?

Reply:
Also, is this directory right place for the testplan to live?  Should it be blacklisted?

If you want to punt this, that is fine, as dealing with testplan stuff is not the point of this PR, and I don't think we need further scope creep.

Reply:
Should indeed be relative